### DISCUSSION TITLE
How the CollapsingFrame control does not expand by default?

### DISCUSSION THREAD

**Discussion by zhenzi0322** [israel-dryer/ttkbootstrap#343] [Q&A]
There may be some menu items that are not expanded when they are initialized. How do I set this up, please?

**Comment by israel-dryer** [comment]
Can you provide a minimal example to demonstrate this issue?

**Reply by zhenzi0322** [reply]
> Can you provide a minimal example to demonstrate this issue?

Thanks, I've solved it
